### DISCUSSION TITLE
The Universe's Expansion May Just Be An Illusion, New Paper Suggests

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#130] [Cosmology Models]
A good 5-minute presentation of the problems with expansion models.
"The Universe's Expansion May Just Be An Illusion, New Paper Suggests"
[https://www.youtube.com/watch?v=TaQBoRLauIY](https://www.youtube.com/watch?v=TaQBoRLauIY)

Then (skip the 1-minute ad) it is suggested that a mathematical transformation can reveal the real universe 😆 🤣 

I left this comment:
> What if someone applied some mathematical transformation to show that the Earth is flat, would the Earth then be flat?
> Physical properties are not changed by a mathematical transformation, this work is just a mathematical exercise with no connection to reality.

**Comment by HanDeBruijn** [comment]
D&eacute;j&agrave; vu all over the place: **[Mathematicists find new glasses](https://github.com/orgs/a-cosmology-group/discussions/63)**.

**Reply by OlliSant** [reply]
The expansion is just an illusion. Can we allready prove that in astronomy? To others?

**Reply by RedshiftDrift** [reply]
@OlliSant how do you prove that "it's an illusion" to someone who does not understand mirrors?

![see through body illusion](https://www.moillusions.com/wp-content/uploads/2015/07/girl-mirror-illusion.jpg)

**Reply by OlliSant** [reply]
I wait when you in ACG make the proof, and then I have the proof to give to others.

The space of the universe does not expand, so it is an illusion, a wrong interpretation of the observations.

I just wait, I hope you make the proof.

**Comment by Francois-Zinserling** [comment]
NYTimes is weighing in on the action ...

[Is There a Crisis in Cosmology ?](https://www.nytimes.com/2023/09/02/opinion/cosmology-crisis-webb-telescope.html)

**Reply by Francois-Zinserling** [reply]
Sorry, didn't know it was a paywall. 

Can I copy/paste the text here?

**Reply by budrap00** [reply]
Regarding the NYT article: https://thisislanduniverse.com/2023/09/05/denial-of-the-deluded/

**Reply by RedshiftDrift** [reply]
@Francois-Zinserling I already attached the paper in the [discussion 139 here](https://github.com/orgs/a-cosmology-group/discussions/139#discussion-5608047).  (It's possible to attach a file by just dragging it in the text window.)

@budrap00 indeed the paper starts well and disappoints later.  The authors still believe in the dark stuff, so they suggest instead something like a review of our ideas about the flow of time... 😮
